app/src/main/kotlin/com/softteco/template/navigation/AppNavHost.kt

```diff
@@ -60,13 +61,17 @@ fun NavGraphBuilder.bottomBarGraph(navController: NavController) {
             LoginScreen(
                 onBackClicked = { navController.navigateUp() },
                 onLoginClicked = {},
-                onSignUpClicked = { navController.navigate(Screen.SignUp.route) }
+                onSignUpClicked = { navController.navigate(Screen.SignUp.route) },
+                onForgotPasswordClicked = { navController.navigate(Screen.ForgotPassword.route) }
             )
         }
         composable(Screen.SignUp.route) {
             SignUpScreen(onBackClicked = { navController.navigateUp() })
         }
     }
+    composable(Screen.ForgotPassword.route) {
```

Comment:
Is there a reason why composable is located outside the nested constructor?

Reply:
fixed